src/components/shared/Counter/Counter.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import styles from "./counter.module.scss";
+import { ButtonIcon } from "../ButtonIcon/ButtonIcon";
+
+interface IPropsCounter {
+  onClick: (value: number) => void;
+}
+
+const Counter = ({ onClick }: IPropsCounter) => {
+  const [counter, setCounter] = useState(1);
+
+  const onCounterIncrease = () => {
+    setCounter(counter + 1);
+    onClick(counter + 1);
+  };
+
+  const onCounterReduce = () => {
+    setCounter(counter - 1);
+    onClick(counter - 1);
+  };
+
+  return (
+    <div className={styles.counterLineWidth}>
+      <ButtonIcon
+        image="https://upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Minus.svg/1200px-Minus.svg.png"
+        description="minus"
+        onClick={onCounterReduce}
+        size="M"
+        disabled={counter <= 1 ? true : false}
```

Comment:
counter <= 1  и так вернет булеан. 

Тернарник тут избыточен и не нужен

```suggestion
        disabled={counter <= 1}
```